### PR TITLE
add a bigint type

### LINKED ISSUE #12: Int64 Support
I'm using Postgres as a backend and `user_id` in my app is defined as Int64 (BigInt). That doesn't match up with `context.session.int()`.

### PR DESCRIPTION
**Pull request by neovintage**
Fixes #12 

/cc @Thyra @sdogruyol

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,14 +58,14 @@ session.ints.each do |k, v|
...
+**BUT:** This should only be used for reading and analyzing values, **never for changing them**. Because otherwise the session won't automatically save the changes and you may produce really weird bugs...
 
-### StorableObject
+#### StorableObject
 
-`kemal-session` has the ability to save objects to session storage. By saving objects to session storage, this opens up the ability to have more advanced data types that aren't supported by the base types (Int32, Float64, String, Bool). 
-Any object that you want to save to session storage needs to be a subclass of `Session::StorableObject`. 
-The subclass needs to define two different methods. First, a class method to deserialize the object from a String, called `unserialize`. The 
-second method, is an instance method called `serialize`. `serialize` will take the object and turn it into a String for the session storage engine to 
+`kemal-session` has the ability to save objects to session storage. By saving objects to session storage, this opens up the ability to have more advanced data types that aren't supported by the base types (Int32, Float64, String, Bool).
```
Comment (sdogruyol) [inline README.md]: need to add `Int64` to base types here

**Comment by sdogruyol** [comment]
@neovintage just a minor comment other than that looks good to me 👍 

WDYT @Thyra ?

File: README.md (hunk 1 above)
Comment (neovintage) [inline README.md]: @sdogruyol fixed

**Comment by sdogruyol** [comment]
LGTM 👍 

Meanwhile we need to update the corresponding engines 💯

**Pull request merged**